packages/cli/ai/baml_src/diff_analyzer.baml

```diff
@@ -58,7 +58,8 @@ function AnalyzeSdkDiff(diff: string @description("The git diff to analyze for g
     - For PATCH: describe the fix or improvement
     - For NO_CHANGE: use type "chore" and state that no functional changes were made
     - Be specific and action-oriented
-    - Always end with the "🌿 Generated with Fern" footer
+    - Always end with the "🌿 Generated with Fern" footer (this may be removed for whitelabel customers)
```

Comment:
let's remove 🌿 Generated with Fern from the prompt altogether, and then we'll add that afterwards on our side